sys/include/net/ieee802154.h

```diff
@@ -32,14 +32,17 @@ extern "C" {
 #endif
 
 /**
- * @brief IEEE 802.15.4 address lengths
```

Comment:
Any reason for these unrelated doc removals?

Reply:
There is nothing removed here, I just inserted `IEEE802154_SFD` above. This is just githubs highlighting :)